starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -217,11 +217,9 @@ async def test_warning_when_max_fee_equals_to_zero(map_contract):
     with pytest.warns(
         DeprecationWarning,
         match=r"Transaction will fail with max_fee set to 0. Change it to a higher value.",
-    ) as max_fee_warnings:
+    ):
         await map_contract.functions["put"].invoke(10, 20, max_fee=0)
 
-    assert len(max_fee_warnings) == 1
```

Comment:
Why do you delete this?

Reply:
Because there are two warnings shown when calling this method, and also I don't think it makes much sense to have this assert if we have `with pytests.warns` anyways.